src/components/UserEdit/index.tsx

```diff
@@ -38,7 +38,7 @@ const messages = defineMessages({
     'Grants auto approval for any requests made by this user.',
   save: 'Save',
   saving: 'Saving...',
-  usersaved: 'User succesfully saved',
+  usersaved: 'User successfully saved',
```

Comment:
We actually already changed this in the `en.json` language file to just be `User Saved`. Let's update this to reflect that.